sql/wsrep_sst.cc

```diff
@@ -1082,6 +1082,8 @@ static uint server_session_execute(MYSQL_SESSION session, std::string query,
                                    bool ignore_error = false) {
   COM_DATA cmd;
   wsp::Sql_resultset rset;
+
+  memset(&cmd, 0, sizeof(cmd));
```

Comment:
I think correctly initializing the 2 new data members in COM_QUERY_DATA would be more explicit here, instead of memsetting everything.

Reply:
It is the same we see in upstream's plugin/sql_service_interface.cc `Sql_service_interface::execute_query`, so I think it is OK.